eng/code-quality-reports/src/main/java/com/azure/tools/checkstyle/checks/Utils.java

```diff
@@ -0,0 +1,46 @@
+package com.azure.tools.checkstyle.checks;
+
+import com.puppycrawl.tools.checkstyle.api.DetailAST;
+import com.puppycrawl.tools.checkstyle.api.TokenTypes;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+
+/**
+ * Common utils amount custom checks
+ */
+public class Utils {
+    private static final Set<Integer> INVALID_FINAL_COMBINATION = new HashSet<>(Arrays.asList(
+        TokenTypes.LITERAL_TRANSIENT,
+        TokenTypes.LITERAL_VOLATILE
+    ));
+
+    private static final Set<String> INVALID_FINAL_ANNOTATIONS = new HashSet<>(Arrays.asList(
```

Comment:
Use `Collections.unmodifiableSet` unless you are expecting people to add to these collections.

Reply:
yes, good point. tks